src/events/http2.js

```diff
@@ -113,16 +114,17 @@ function httpWrapper(wrappedFunction, authority) {
                 });
 
                 clientRequest.once('close', () => {
-                    setJsonPayload(httpEvent, 'response_body', data);
+                    setJsonPayload(httpEvent, 'response_body', Buffer.concat(chunks), responseHeaders['content-encoding']);
```

Comment:
what happens if the response doesn't have a `content-encoding` header?

Reply:
It will be `undefined` which means that the code will not try to decode it.